tests/positive/sync_val.cpp

```diff
@@ -13,9 +13,9 @@
 
 #include "../framework/layer_validation_tests.h"
 
-class VkPositiveSyncValTest : public VkSyncValTest {};
+class kPositiveSyncVal : public VkSyncValTest {};
```

Comment:
```suggestion
class PositiveSyncVal : public VkSyncValTest {};
```
I'm guessing that was the intention?

Reply:
........yes